### PR TITLE
[WIP] Adding abstraction

### PR DESCRIPTION
**Pull request by ehsteve**
The purpose of this merge request is to abstract out our instrument and mission names so that other missions could use this. I also was able to optimize some of the code. 

**This is a work in progress.**

### CODE REVIEW

**Comment by ehsteve** [comment]
Open tasks

- [ ] Could not figure out how to programmatically import hermes.log
- [ ] Not sure i understand how level upgrade occurs. Looks like data levels are hard coded which is bad practice.

**Comment by ehsteve** [comment]
@dbarrous how do you want to proceed with this PR?

**Comment by dbarrous** [comment]
> @dbarrous how do you want to proceed with this PR?

@ehsteve I'm currently trying to abstract it a bit more so that we just have a singular `vars.py` file that we can modify with the mission name, mission package, and instrument name instead. 

For the open tasks on the levels, I'm going to import them from the mission package since that makes the most sense, and that way it isn't hard-coded into the processing module. Also still trying to figure out how to import hermes.log programmatically as well.

File: lambda_function/src/util.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+"""
```
Comment (ehsteve) [inline lambda_function/src/util.py:1]: this is a small amount of code. why not keep it in the lambda? that way it is clearer what is happening.

File: lambda_function/src/vars.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Mission name
```
Comment (ehsteve) [inline lambda_function/src/vars.yaml:1]: can this file be called something more descriptive like config.yaml. vars could be anything.

**Pull request merged**